db/seeds.rb

```diff
@@ -72,6 +73,15 @@
   ubs.save!
 end
 
+User.new.tap do |user|
+  user.name = 'admin'
+  user.password = 'dontpanic'
+  user.password_confirmation = 'dontpanic'
+  user.administrator = true
+  user.ubs = Ubs.all
+  user.save!
+end
+
 [
```

Comment:
```suggestion
end

User.all.each do |user|
  if user.ubs.blank?
    ubs_user = Ubs.find_by(user: user)
    user.update!(ubs: [ubs_user])
  end
end

[
```

Um detalhe que percebi que poderia dar problema com a configuração dos usuários "legados" (não logados rs) é o relacionamento `user.ubs` retornar um array vazio, por isso a sugestão dessa verificação e update nos `users` que não possuem este relacionamento, para não dar o erro abaixo quando este operador logar na aplicação por exemplo:

![Captura de tela de 2021-04-25 05-26-42](https://user-images.githubusercontent.com/19494561/115987162-32209c80-a58a-11eb-865d-dd5f92a2cc86.png)

Reply:
![Screen Shot 2021-04-25 at 12 11 32](https://user-images.githubusercontent.com/3586/115998805-66fb1680-a5bf-11eb-92ca-bd01249d21ef.png)

Boa, bem observado. Estou tratando este caso nesta tela, e arrumei as seeds para evitar que isso ocorra.